production/ksonnet/loki/query-scheduler.libsonnet

```
@@ -0,0 +1,58 @@
+
+local k = import 'ksonnet-util/kausal.libsonnet';
+
+{
+  local container = k.core.v1.container,
+
+  // Override frontend and querier configuration
+  local override = {
+      frontend: {
+          scheduler_address: 'query-scheduler.%s.svc.cluster.local:9095' % $._config.namespace,
+      },
+      frontend_worker: {
+          frontend_address: '',
+          scheduler_address: 'query-scheduler.%s.svc.cluster.local:9095' % $._config.namespace,
+      },
+  },
+  _config +: {
+      loki: std.mergePatch($._config.loki, override),
```

This should be equivalent

```suggestion
      loki+: override,
```

---

I looked it up and according to [Stackoverflow](https://stackoverflow.com/questions/61643489/what-is-the-difference-between-the-operator-and-std-mergepatch-in-jsonnet) `+:` would remove everything in the path.

---

This should be easy to test, but we've used this pattern historically without said problem, for instance: https://github.com/grafana/loki/blob/main/production/ksonnet/loki/index-gateway.libsonnet#L7-L16